conf/config.inc.php

```diff
@@ -168,6 +168,9 @@
 # What attribute should be changed by the changesshkey action?
 $change_sshkey_attribute = "sshPublicKey";
 
+# Ensure the SSH Key submitted uses a type we trust
+$ssh_valid_key_types = array('ssh-rsa', 'ssh-dss', 'ecdsa-sha2-nistp256', 'ssh-ed25519');
```

Comment:
Can you please add ecdsa-sha2-nistp384 ecdsa-sha2-nistp521

Reply:
ok